examples/aggregator/module/rust/src/lib.rs

```diff
@@ -174,19 +182,59 @@ pub struct Config {
 
 oak::entrypoint!(oak_main => |in_channel| {
     oak::logger::init_default();
+
+    // Parse config.
     let config_map = oak::app_config_map(in_channel).expect("Couldn't read config map");
     let config: Config = toml::from_slice(&config_map.items.get("config").expect("Couldn't find config")).expect("Couldn't parse TOML config file");
     info!("Parsed config: {:?}", config);
-    let grpc_channel =
-        oak::grpc::server::init(&config.grpc_server_listen_address).expect("could not create gRPC server pseudo-Node");
-    let (invocation_sender, invocation_receiver) =
-        oak::io::channel_create::<oak::grpc::Invocation>()
-            .expect("could not create gRPC invocation channel");
-    oak::node_create(
-        &oak::node_config::wasm("app", "grpc_worker"),
-        invocation_receiver.handle,
-    ).expect("could not create gRPC worker node");
-    while let Ok(invocation) = grpc_channel.receive() {
+    let grpc_server_channel =
+        oak::grpc::server::init(&config.grpc_server_listen_address).expect("Couldn't create gRPC server pseudo-Node");
+
+    // Create an Aggregator Node.
+    let (init_sender, init_receiver) = oak::io::channel_create::<AggregatorInit>()
+        .expect("Couldn't create initialization channel");
+    oak::node_create(&oak::node_config::wasm("app", "aggregator"), init_receiver.handle)
+        .expect("Couldn't create gRPC worker node");
+    oak::channel_close(init_receiver.handle.handle).expect("Couldn't close receiver channel");
+
+    // Create a gRPC client Node with a less restrictive label than the current Node.
+    // In particular, the confidentiality component of the current Node label includes the
+    // "aggregator" WebAssembly hash, which is declassified as part of the gRPC client
+    // Node creation. This is only allowed if the current Node actually has the
+    // appropriate capability (i.e. the correct WebAssembly module hash) as specified by
+    // the label component being removed, as set by the client.
+    // TODO(#814): Uncomment and use correct confidentiality labels.
+    // let label = Label {
+    //     confidentiality_tags: vec![Tag {
+    //         tag: Some(tag::Tag::TlsEndpointTag(TlsEndpointTag {
+    //             certificate_subject_alternative_name: "google.com".to_string(),
+    //         })),
+    //     }],
+    //     integrity_tags: vec![],
+    // };
+    let grpc_client = oak::grpc::client::Client::new_with_label(
+        &oak::node_config::grpc_client("https://localhost:8888"),
+        &Label::public_untrusted(),
+    )
+    .expect("Could not create a gRPC client");
+
+    // Send the initialization message to Aggregator Node containing a gRPC server invocation
+    // receiver and a gRPC client invocation sender.
+    debug!("Sending the initialization message message to handler Node");
```

Comment:
```suggestion
    debug!("Sending the initialization message to handler Node");
```
or is it actually a message containing an initialization message? :)